package.json

```diff
@@ -119,9 +118,11 @@
     "qunit": "2.20.1",
     "raw-body": "2.5.2",
     "requirejs": "2.3.6",
-    "rollup": "2.79.1",
+    "rollup": "4.12.0",
```

Comment:
iirc, this won't work on Node 10. I'm working on migrating git builds to GH actions, so maybe that will need to come first.

Reply:
Yes, possibly. Otherwise, I’ll need to split it, creating a separate aliased version for tests - and I’m not even sure if that’s going to work given binaries will still conflict.

Reply:
This change no longer exists as the version was updated on `main`.